src/Prometheus.Client.MetricServer/Prometheus.Client.MetricServer.csproj

```diff
@@ -5,8 +5,8 @@
     <AssemblyTitle>Prometheus.Client.MetricServer</AssemblyTitle>
     <VersionPrefix>2.1.1</VersionPrefix>
     <Authors>Sergey Kuznetsov</Authors>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.3;netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net45;netstandard1.3;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.3;netstandard2.0;netcoreapp2.2;</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net47;netstandard1.3;netstandard2.0;netcoreapp2.2;</TargetFrameworks>
```

Comment:
Do we need this target? As far as we have netstandard2.0

Reply:
Just coming back to .net after a while away and im not sure!

I will take your lead on this and remove it!